modules/app-configuration/configuration-store/.test/encr/main.test.bicep

```diff
@@ -0,0 +1,98 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.appconfiguration.configurationstores-${serviceShort}-rg'
```

Comment:
```suggestion
param resourceGroupName string = 'dep--${namePrefix}-appconfiguration.configurationstores-${serviceShort}-rg'
```

Reply:
The other files should be updated by @eriqua's PR

Reply:
@AlexanderSehr done :)